lib/services/p2p/index.js

```diff
@@ -168,10 +170,17 @@ P2P.prototype._broadcast = function(subscribers, name, entity) {
 };
 
 P2P.prototype._connect = function() {
-  this._connectCalled = this._connectCalled > 0 ? 2 : 1;
-  if (this._connectCalled > 1 || !this._bcoin) {
+  var self = this;
+  self._connectCalled = self._connectCalled > 0 ? 2 : 1;
+  if(self._connectCalled > 1 || !self._bcoin) {
     log.info('Connecting to p2p network.');
-    this._pool.connect();
+    self._pool.connect();
+    var retryInterval = setInterval(function() {
```

Comment:
Is it possible to drop this polling if self._pool.connect() were added to the Bcoin start callback?